src/printer.ts

```diff
@@ -773,6 +775,20 @@ export class PugPrinter {
 			if (token.val !== true) {
 				this.result += `=${token.val}`;
 			}
+		} else if (token.name === 'class' && this.options.pugClassNotation === 'attribute') {
+			const val: string = isQuoted(token.val) ? token.val.slice(1, -1).trim() : token.val;
+			const classes: string[] = val.split(/\s+/);
```

Comment:
**thought:** I'm a little bit afraid of this line and don't know what it's doing if the class contains js code with spaces 🤔  
But I think I will merge it for now